src/types/options.ts

```diff
@@ -98,11 +99,17 @@ interface Options {
      * Label displayed on the submit button.
      */
     submitLabel?: string;
+    titleLabel?: string;
+    totalLabel?: string;
+    currencyBehind?: boolean;
+    showTotal?: boolean;
   };
   /**
    * Legend visibility.
    */
   legendVisible?: boolean;
+  reservedLegendLabel?: string;
+  hideLegendPrice?: boolean;
```

Comment:
As mentioned above this would become:

```
/**
 * Legend options.
 */
legend: {
  /**
   * Legend visibilty.
   */
  visible: boolean;
  priceVisible: boolean;
  reservedLabel: string;
}
```

If you could document the remaining properties as well it would be great!